core/src/raw/oio/list/flat_list.rs

```diff
@@ -92,7 +94,7 @@ where
     async fn next(&mut self) -> Result<Option<oio::Entry>> {
         loop {
             if let Some(de) = self.next_dir.take() {
-                let (_, l) = self.acc.list(de.path(), OpList::new()).await?;
+                let (_, l) = self.acc.list(de.path(), self.args.clone()).await?;
```

Comment:
It's a bit tricky since `FlatLister` is used to simulate recursive listing on unsupported services. Therefore, we can't forward `OpList` directly. Instead, let's introduce a `start_after` here, and `FlatLister` should ensure the `OpList` passed to undering are constructed correctly.

Reply:
meaning `OpList::new()` with `start_after` set to a `self.start_after` value? or am I misinterpreting?

Reply:
> meaning `OpList::new()` with `start_after` set to a `self.start_after` value?

Yep.